src/Build/BackEnd/Node/ServerNodeBuildCommand.cs

```diff
@@ -49,20 +52,50 @@ internal sealed class ServerNodeBuildCommand : INodePacket
         /// </summary>
         public CultureInfo UICulture => _uiCulture;
 
+        /// <summary>
+        /// Buffer width of destination Console.
+        /// Console loggers are supposed, on Windows OS, to be wrapping to avoid output trimming.
+        /// -1 console buffer width can't be obtained.
+        /// </summary>
+        public int ConsoleBufferWidth => _consoleBufferWidth;
```

Comment:
minor: why not just 
`public int ConsoleBufferWidth { get; }` & instantiating this,  instead of referencing a field in this property?

Reply:
Unfortunately backing field is need in this as it is used for set/get in de/ser.

Reply:
See method Translate